web/src/controllers/Scripts.ts

```diff
@@ -48,6 +53,22 @@ export class Installers {
     }
     installer = installer.resolve();
 
+    try {
+      let clientIP = request.get("True-Client-IP");
```

Comment:
How about using this here https://www.npmjs.com/package/request-ip

Reply:
good idea